app/controllers/booking_requests_controller.rb

```diff
@@ -5,17 +5,17 @@ class BookingRequestsController < ApplicationController
   before_action :set_breadcrumbs
 
   def step_one
-    @booking_feedback = BookingFeedbackForm.new
+    @feedback = FeedbackForm.new(booking_type: 'online_booking')
```

Comment:
Is there any mileage in defaulting this argument to 'online_booking' so we don't have to specify it everywhere? It's overwhelmingly the normal value passed. Or perhaps a specialised method that initialises the object with that `booking_type`. Perhaps `FeedbackForm.for_online_booking` or something?